### PR TITLE
Server info proper values

### PR DESCRIPTION
**Pull request by flovilmart**

### CODE REVIEW

File: src/ParseServer.js [hunk 1]
```diff
@@ -181,9 +181,10 @@ class ParseServer {
     // We pass the options and the base class for the adatper,
     // Note that passing an instance would work too
     const pushController = new PushController();
-
-    const hasPushSupport = pushAdapter && push;
-    const hasPushScheduledSupport = pushAdapter && push && scheduledPush;
+    const hasPushSupport = (pushAdapter && push) != undefined;
+    const hasPushScheduledSupport = hasPushSupport && (scheduledPush == true);
+    //const hasPushSupport = pushAdapter && push;
```
Comment (acinader) [inline src/ParseServer.js]: remove

File: src/ParseServer.js [hunk 2]
```diff
@@ -181,9 +181,10 @@ class ParseServer {
     // We pass the options and the base class for the adatper,
     // Note that passing an instance would work too
     const pushController = new PushController();
-
-    const hasPushSupport = pushAdapter && push;
-    const hasPushScheduledSupport = pushAdapter && push && scheduledPush;
+    const hasPushSupport = (pushAdapter && push) != undefined;
+    const hasPushScheduledSupport = hasPushSupport && (scheduledPush == true);
```
Comment (acinader) [inline src/ParseServer.js]: prefer `===` and '!==='

Comment (acinader) [inline src/ParseServer.js]: whoa.  i just tested `undefined && undefined` in the node repl.  never would have guessed that.  nice.

**Comment by flovilmart** [comment]
Yeah that's a bit silly!

**Pull request merged**